views/home.pug

```diff
@@ -1,2 +1,7 @@
 h1 KaaBee
 h2 Welkom, wij hebben #{ storeCount } winkels die staat te popelen om je te ontvangen!
+ul
+    li
```

Comment:
Ik heb tot nu toe 2 spaties gebruikt als indentatie voor pug. Willen we daar vier van maken?